lib/solidus_friendly_promotions/configuration.rb

```diff
@@ -107,6 +107,12 @@ def initialize
       default: 1,
       post: 2
     }
+
+    preference :use_new_admin, :boolean, default: false
+
+    def use_new_admin?
+      SolidusSupport.admin_available? && preferred_use_new_admin
+    end
```

Comment:
This preference has two uses: It can be used to force the new admin controllers for people who wish to do that, and, more importantly, it can be used to enable the new admin controllers from the test suite. Relying on cookies or parameters is very hard in feature specs, which is what we have for the new admin (cookie management is driver-dependent, the driver changes between `before` blocks, and generally we want the app, not the tests to set cookies).